src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Repository/ServiceRepository.php

```diff
@@ -18,6 +18,7 @@
 
 namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Repository;
 
+use Doctrine\DBAL\Exception;
```

Comment:
:wastebasket: ?

Reply:
Will be removed in https://github.com/SURFnet/sp-dashboard/commit/65999c53